### ISSUE TITLE
The query /metrics/find/?query=&format=completer fails

### ISSUE THREAD

**Issue by azhiltsov** [go-graphite/carbonapi#312]
This query is perfectly valid and should return Top Level directory list, like original graphite does.
It being used in http://graphite-web/dashboard/ 
Hovewer carbonapi can't serve such queries with  "missing parameter `query`" error
```
$ curl -X POST 'http://carbonapi/metrics/find/?query=&format=completer'
missing parameter `query`
```

I discovered that query=* is doing what expected in carbonapi.
But in original graphite you can do query=. query=/ query=* and query= 
So carbonapi should probably mimic the same behaviour.

**Comment by azhiltsov** [comment]
carbonapi in th question  13ebf4a

**Comment by azhiltsov** [comment]
should be fixed #327 #320 #322
